src/Standard/LocalWeaponSetup/init.luau

```diff
@@ -21,11 +21,17 @@ local LocalWeaponSetup = {}
 
 export type LocalWeaponSetup = typeof(LocalWeaponSetup)
 
+local AlreadySetupTools = {} :: {Tool}
```

Comment:
This can be made *slightly* more efficient if this is treated as a map (`{[Tool]: boolean}`), but just *slightly*. Line 31-33 would become `if not AlreadySetupTools[Tool] then return end` and line 107 would become `AlreadySetupTools[Tool] = true`. I believe we do this in all cases currently and `table.find` is unique.

However, this approach is fine. Lines 31-33 could be changed to 1 line like all of the other "return early" cases, though.